system/jlib/jtrace.cpp

```diff
@@ -1857,8 +2048,21 @@ ISpan * CTraceManager::createServerSpan(const char * name, const IProperties * h
     return new CServerSpan(name, httpHeaders, flags, spanStartTimeStamp);
 }
 
-ISpan * createBackdatedInternalSpan(const char * name, stat_type elapsedNs)
+ISpan * createActiveConditionalInternalSpan(const char * name, stat_type thresholdNs)
+{
+    return queryThreadedActiveSpan()->createConditionalInternalSpan(name, thresholdNs);
+}
+
+ISpan * createActiveConditionalClientSpan(const char * name, stat_type thresholdNs)
+{
+    return queryThreadedActiveSpan()->createConditionalClientSpan(name, thresholdNs);
+}
+
+ISpan * createBackdatedInternalSpan(const char * name, stat_type elapsedNs, stat_type thresholdNs)
```

Comment:
this approach doesn't seem too helpful, but centralizes the logic for conditionally reporting a backdatedinternalspan...